### ISSUE TITLE
Staging Review finding: Button - Loading imposter component

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#114349]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Lifestage Pension and Burial
**Product:** Pension and Burial
**Feature:** Income and Asset Statement 21P-0969

## Findings details
**VA.gov Experience Standard - issue:** [User encounters components and patterns that meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (QA), @rsmithadhoc (Accessibility)

### Description

**Submit Application/Sending.../Submitted** can be updated to use [Button - Loading](https://design.va.gov/components/button/#loading). This button also does not have a sufficient contrast ratio between the text and background, which would be fixed by using the VADS button.

<details><summary>Link, screenshot or steps to recreate</summary>

https://staging.va.gov/income-and-asset-statement-form-21p-0969/review-and-submit

<img width="636" height="95" alt="Image" src="https://github.com/user-attachments/assets/83cb91b4-8155-41af-a84d-f887e953b223" />

</details>

## Recommended action

Use the VADS component.

## Acceptance Criteria

- [x] Submission pending button includes in-progress animation
- [x] Disabled submission pending button contrast is satisfactory
- [x] Disabled submitted button contrast is satisfactory
- [x] All tests passing

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.4.3 Contrast (Minimum) - Level AA](https://www.w3.org/TR/WCAG22/#contrast-minimum)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jerekshoe** [comment]
@shiragoodman This is part of the forms system. Shouldn't the form system team be updating this or are we expected to?

**Comment by Mottie** [comment]
I'll pick up this one since it's related to the form navigation imposter component work I've been doing.

**Comment by Mottie** [comment]
This work is now available in staging for testing & verification.

**Comment by Mottie** [comment]
Ticket is ready to close.
